.github/workflows/publish_packages.yaml

```
@@ -0,0 +1,27 @@
+name: publish_packages
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)